doc/tor.md

```diff
@@ -211,3 +209,100 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+  Another option for Bitcoin network privacy is to use I2P (option `i2p`).
+
+- For maximum privacy, it is preferable to disable listening instead of running an
+  onion service.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  The `onlynet=onion` configuration option can potentially ensure the node only attempts to connect over
+  the Tor network. The decision to use this option should consider the following trade-offs:
+
+  Only connecting over Tor is more private when you combine it with no reachable IPv4/IPv6 address,
+  in particular if you want to broadcast transactions without them being correlatable with your IP.
+
+  **Sybil Attacks**
+
+  On the other hand, if you only make random Tor connections, you're much more vulnerable to Sybil attacks.
+  As Tor addresses may be created at no cost, an attacker can potentially flood the network with many Tor
+  nodes and receive all of the outbound Tor connections an `onlynet=tor` node makes.
+  This is significantly less a concern with IPv4/IPv6 (especially with asmap) due to cost of getting IPs in many
+  networks. It's also less a concern if you have `-addnode` connections to trusted peers (even if they're onion addresses).
+
+  **Network Partitioning**
+
+  If too many nodes use `onlynet=onion`, it could become difficult for onion nodes to communicate with clearnet nodes.
+  It is essential that some nodes access both clearnet and Tor or use `onlynet=i2p` with `onlynet=onion`.
+
+## Setup Guide: Debian/Ubuntu
+
+Assuming you already have an installation of Bitcoin Core:
+
+**1. Install Tor:**
+
+To install Tor, run:
+
+```
+sudo apt install tor
+```
+
+**2. Edit Tor configuration file:**
+
+Open the `torrc` file with your text editor. This example uses nano:
+
+```
+sudo nano /etc/tor/torrc
+```
+Ensure that `torrc` has these settings, and save:
+
+```
+ControlPort 9051
+CookieAuthentication 1
+CookieAuthFileGroupReadable 1
+```
+
+**3. Add user to Tor group ("satoshi" in this example):**
+
+```
+sudo usermod -a -G debian-tor satoshi
+```
+
+**4. Restart System**
+
+```
+sudo reboot
+```
+
+**5. Edit Bitcoin configuration file**
+
+Open the `bitcoin.conf` file with your text editor.
```

Comment:
This should probably at least link to instructions about where to find your configuration directory, e.g. https://en.bitcoin.it/wiki/Running_Bitcoin#Bitcoin.conf_Configuration_File

(Maybe an idea would be to have `bitcoin -version` or another option that can be run concurrent with the main program report the full path to the various used directories.)

Reply:
> This should probably at least link to instructions about where to find your configuration directory, e.g. en.bitcoin.it/wiki/Running_Bitcoin#Bitcoin.conf_Configuration_File

Done. Made changes in https://github.com/bitcoin/bitcoin/commit/16db258eafa21328b9433c2da6dce6fbc8f532de

> (Maybe an idea would be to have bitcoin -version or another option that can be run concurrent with the main program report the full path to the various used directories.)

Maybe can add information about [files](https://github.com/bitcoin/bitcoin/blob/master/doc/files.md) in `bitcoind -help` results?